var/spack/repos/builtin/packages/iegenlib/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# -----------------------------------------------------------------------------
+# Author: Derick Huth <derick.huth@utah.edu>
```

Comment:
Can you add yourself in the `maintainers` variable inside the package?  Grep for examples...